architecture/additional_concepts/authorization.adoc

```diff
@@ -228,17 +228,192 @@ TIP: Remember that xref:authentication.adoc#users-and-groups[users
+ 
+|watch
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints persistentvolumeclaims replicationcontrollers replicationcontrollers/scale secrets serviceaccounts services services/proxy bindings events limitranges namespaces namespaces/status pods/log pods/status replicationcontrollers/status resourcequotas resourcequotas/status horizontalpodautoscalers cronjobs jobs scheduledjobs deployments deployments/rollback deployments/scale horizontalpodautoscalers jobs replicasets replicasets/scale replicationcontrollers/scale daemonsets statefulsets rolebindings roles policies policybindings rolebindingrestrictions buildconfigs buildconfigs/webhooks builds builds/log deploymentconfigs deploymentconfigs/scale generatedeploymentconfigs deploymentconfigs/log deploymentconfigs/status imagestreamimages imagestreammappings imagestreams imagestreams/secrets imagestreamtags imagestreams/status appliedclusterresourcequotas routes routes/status processedtemplates templateconfigs templates buildlogs resourcequotausages 
+ 
+|===
+
+.basic-user Rules Matrix
+[cols="2s,8l"]
+|===
+|create
+|selfsubjectrulesreviews localsubjectaccessreviews subjectaccessreviews 
+ 
+|get
+|users clusterroles 
+ 
+|list
+|projectrequests clusterroles storageclasses projects 
+ 
+|watch
+|projects 
+ 
+|===
+
+.cluster-admin Rules Matrix
+[cols="2s,8l"]
+|===
+
+|*
+|*
+
+|===
+
+.edit Rules Matrix
+[cols="2s,8l"]
+|===
+|create
+|pods pods/attach pods/exec pods/portforward pods/proxy configmaps endpoints persistentvolumeclaims replicationcontrollers replicationcontrollers/scale secrets serviceaccounts services services/proxy horizontalpodautoscalers cronjobs jobs scheduledjobs deployments deployments/rollback deployments/scale horizontalpodautoscalers jobs replicasets replicasets/scale replicationcontrollers/scale statefulsets buildconfigs buildconfigs/webhooks builds buildconfigs/instantiate buildconfigs/instantiatebinary builds/clone deploymentconfigs deploymentconfigs/scale generatedeploymentconfigs deploymentconfigrollbacks deploymentconfigs/instantiate deploymentconfigs/rollback imagestreamimages imagestreammappings imagestreams imagestreams/secrets imagestreamtags imagestreamimports routes processedtemplates templateconfigs templates buildlogs 
```

Comment:
All these values appear in a single scrollable line when rendered, it will make sense to break all these into separate lines for better readability.